### PR TITLE
KOGITO-5203 Validate use of onEntry and onExit scripts

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/canonical/ProcessToExecModelGenerator.java [hunk 1]
```diff
@@ -95,23 +95,6 @@ public ProcessMetaData generate(WorkflowProcess process) {
         return metadata;
     }
 
-    public MethodDeclaration generateMethod(WorkflowProcess process) {
```
Comment (tiagodolphine) [inline jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/canonical/ProcessToExecModelGenerator.java:98]: this method was not being used?

File: kogito-codegen-modules/kogito-codegen-processes/src/main/java/org/kie/kogito/codegen/process/ProcessExecutableModelGenerator.java [hunk 2]
```diff
@@ -55,21 +55,16 @@ public String description() {
...
             generate();
+        }
         return processMetaData.getProcessClassName();
     }
 
     public String generatedFilePath() {
         return processFilePath;
     }
 
-    private String getCompiledClassName(Path fileNameRelative) {
```
Comment (tiagodolphine) [inline kogito-codegen-modules/kogito-codegen-processes/src/main/java/org/kie/kogito/codegen/process/ProcessExecutableModelGenerator.java:67]: this method was not being used?

**Review by tiagodolphine — APPROVED** [review]
LGTM

File: jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/canonical/ProcessToExecModelGenerator.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm/jbpm-flow-builder/src/main/java/org/jbpm/compiler/canonical/ProcessToExecModelGenerator.java:98]: no

File: kogito-codegen-modules/kogito-codegen-processes/src/main/java/org/kie/kogito/codegen/process/ProcessExecutableModelGenerator.java (hunk 2 above)
Comment (cristianonicolai) [reply kogito-codegen-modules/kogito-codegen-processes/src/main/java/org/kie/kogito/codegen/process/ProcessExecutableModelGenerator.java:67]: nope

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by ricardozanini — APPROVED** [review]
Do we have a JIRA to track onEntry/OnExit scripts? Or are we going to use [KOGITO-2067](https://issues.redhat.com/browse/KOGITO-2067)?
Either way, I've linked both JIRAs.

**Pull request merged**